src/pudl/metadata/classes.py

```diff
@@ -1466,7 +1466,7 @@ def format_df(self, df: pd.DataFrame | None = None, **kwargs: Any) -> pd.DataFra
                 and pd.api.types.is_integer_dtype(df[field.name])
             ):
                 df[field.name] = pd.to_datetime(df[field.name], format="%Y")
-            if pd.api.types.is_categorical_dtype(dtypes[field.name]):
+            if isinstance(dtypes[field.name], pd.CategoricalDtype):
```

Comment:
deprecation warning # 2 (the loud one omigosh so many warnings shhhhh)

Reply:
Oh I fixed this one somewhere too. Yeah it was loud.